backend/entrypoint.fuzz.sh

```
@@ -43,7 +43,7 @@ fi
 echo "Starting fuzzing process..."
 
 if [ -n "$CI" ]; then
-    pytest ./tests/${TEST_FILE}
+    poetry run pytest ./tests/${TEST_FILE}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)